tool.c

```diff
@@ -387,6 +453,7 @@ static const struct tool tools[] = {
 	{"compdb", "dump compilation database", compdb},
 	{"query", "show incoming/outgoing edges for a path", query},
 	{"targets", "list targets", targets},
+	{"graph", "output graphviz dot file for targets", graphviz},
```

Comment:
Could you sort this by tool name? Also, try to order the functions in the same order they appear in this list. I know `query` and `targets` are currently out of order, but I'll fix this after merging this PR.